pystac_client/item_search.py

```diff
@@ -31,7 +41,7 @@
 #         ...
 
 DatetimeOrTimestamp = Optional[Union[datetime_, str]]
-Datetime = Union[Tuple[str], Tuple[str, str]]
```

Comment:
this change simply reflects what the method that returned it was already doing.  The datetime parameter to STAC API Search is a string, not a tuple, so this aligns with that when generating the request params